### DISCUSSION TITLE
Font, spacing between letters and between lines

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#10310] [General]
In fheroes2, the letters are tighter and the line spacing is larger compared to the original game.
This can be easily noticed with the scenario description which is not displayed in the same way.

In the original game:

![Scenario selection 01](https://user-images.githubusercontent.com/43583994/90317136-941f0180-df27-11ea-91ef-c9512be6614d.png)

In fheroes2:

![Scenario selection 02](https://user-images.githubusercontent.com/43583994/90317151-abf68580-df27-11ea-9277-ee9027b55b33.png)

**Comment by Branikolog** [comment]
Hi, @LeHerosInconnu!
I can see the difference. But doesn't it looks better in fheroes2, in your opinion?

**Comment by LeHerosInconnu** [comment]
Hello @Branikolog,
I'm so used to playing with the airier font from the original Heroes 2 that the texts in fheroes2 seem strange to me.
I have the impression that the text is all curled up on itself, it gives a feeling of oppression.
And it is even more obvious, for example in small info windows, I have the impression that the text has been compressed to be able to fit into the window space.

**Comment by a1exsh** [comment]
Additionally, the right margin in the description box seems to be non-existent, should be at least one pixel:
![image](https://user-images.githubusercontent.com/489601/138455020-ff2efa7f-84e9-442d-888a-c746c2a0b25c.png)

**Comment by a1exsh** [comment]
Also the bottom margin is missing:
![image](https://user-images.githubusercontent.com/489601/138455592-f58a8408-e164-412f-a242-411128bba6dc.png)

**Comment by ihhub** [comment]
I am moving this issue out of the original project scope. While it is good to replicate the original game by 100% the current implementation allows us to have more text to be displayed and players are used to it.

**Comment by ihhub** [comment]
Moving this issue to the Discussion section as I don't think we are going to use the original text alignment.
